src/AcceptanceTests.SQS/ConfigureSQSTransportTestExecution.cs

```diff
@@ -14,9 +14,19 @@ public BridgeTransportDefinition GetBridgeTransport()
     {
         var transportDefinition = new TestableSQSTransport(NamePrefixGenerator.GetNamePrefix());
 
+        var bridgeTransportConfiguration = new BridgeTransportConfiguration(transportDefinition);
+
+        bridgeTransportConfiguration.RegisterAddressParser(address =>
+        {
+            var endpointNameWithoutPrefix = address.Split(NamePrefixGenerator.Separator).Last();
+            var endpointNameWithWithReversedNamingConvention = endpointNameWithoutPrefix.Replace("-", ".");
+
+            return endpointNameWithWithReversedNamingConvention;
```

Comment:
@TravisNickels @kbaley this reverses the naming convention + prefix use to make sure that the addresses match

Users would have to do the same thing if they use a prefix or if they are using _ in their endpoint names